### ISSUE TITLE
Temperature not matching when `format=` is used

### ISSUE THREAD

**Issue by rieje** [chubin/wttr.in#705]
curl 'wttr.in/<GPS coordinates>'
    curl 'wttr.in/<same GPS coordinates>?format=%c+%C+%m+%w+%t+%f'

The former reports `24(21) °F` and the latter reports `☀️   Clear 🌒 0mph +24°F +20°F`. I'm not sure how the "feels like" temperature is not the same (21°F vs 20°F). I've run the commands multiple times at the same time and this is consistent.

**Comment by chubin** [comment]
I am not sure about it. I rather would expect that it is a caching problem, and some of the queries are handled
from the cache, and some are not. To prove it, please append `&nonce=$RANDOM` to the query, like this:

```
curl wttr.in/<LOCATION>?format=<FORMAT>&nonce=$RANDOM
```

**Comment by rieje** [comment]
`curl 'wttr.in/<GPS coordinates>' ` does not support `&nonce=$RANDOM` so it's not cached and always accurate? 

Anyway, the temperatures are still different (same temperatures reported with

    curl "wttr.in/<LOCATION>?format=<FORMAT>&nonce=$RANDOM"

and
 
    curl "wttr.in/<LOCATION>?format=<FORMAT>"

**Comment by Blisk** [comment]
I also niticed that temperature is differend from what I see online sometimes 3 degrees

**Comment by chubin** [comment]
Theoretically, it is possible that the location is resolved to wrong coordinates.
To verify that, specify `format=j1` as an argument, and check the coordinates of the `nearest_area`

**Comment by ronnyworm** [comment]
Unfortunately `&nonce=$RANDOM` does not produce the correct output either:
```
curl "wttr.in/Berlin?Q0&nonce=$RANDOM&lang=de&format=3"
Berlin: ☀️   +20°C

curl "wttr.in/Berlin" | head -n 5  
Weather report: Berlin

      \   /     Sunny
       .-.      +29(28) °C     
    ― (   ) ―   ↖ 9 km/h   
```

I verified that the coordinates are the same. Is there a different way to clear the cache?

**Comment by Blisk** [comment]
Yes I tested that too and still temperature varies alot from 2-4 oC and location is valid.

**Comment by VuillaumeXB** [comment]
Yes, this issue still persists.

**Comment by chubin** [comment]
I believe the main issue we face here is related to caching. We simply need more capacity and upstream data to reduce the caching time.

Additionally, you can bypass the second layer of caching by switching the language setting, for example, using `lang=fr` instead of `lang=de`.

I understand this might sound a bit unusual, but until the capacity issue is resolved, such tricks and workarounds can occasionally be helpful.

**Comment by VuillaumeXB** [comment]
Will try that workaround in the meantime. Thank you.

**Comment by c0GZ** [comment]
I tried ``format=j1`` to see the json and the problem seems to be that it shows the time it was 8am, which is not applicable during the whole day...

**Comment by zackb** [comment]
> What I found that worked was to set the language to a language that doesn't exist, ie `lang=gibberish`

This works great
